controllers/userController.js

```diff
@@ -8,6 +8,7 @@ const User = require("../models/User");
 const createHttpError = require("../utils/createHttpError");
 const { USER } = require("../constants/responseMessages");
 const { SIGNUP, SERVER } = require("../constants/error");
+const { RESULT } = require("../constants/responseMessages");
```

Comment:
```suggestion
const { USER, RESULT } = require("../constants/responseMessages");
const { SIGNUP, SERVER } = require("../constants/error");
```
자동 완성으로 인해서 분리된 것 같습니다